store/modules/events.ts

```diff
@@ -15,7 +16,7 @@ export { CommunityEvent, CommunityEventType, WorldLocation }
 
 export default {
   state: {
-    activeTab: 0,
+    communityEventsShowed: 'upcoming',
```

Comment:
I like `upcoming` although, in the collection, we use `future`. Can you use the same for both? I don't mind which of them.

Reply:
I'm going to use `upcoming`